xbmc/cores/VideoPlayer/VideoRenderers/VideoShaders/WinVideoFilter.cpp

```diff
@@ -1116,12 +1127,7 @@ void CConvolutionShaderSeparable::SetShaderParameters(CD3DTexture &sourceTexture
   m_effect.SetTexture( "g_Texture",  sourceTexture );
   m_effect.SetTexture( "g_KernelTexture", m_HQKernelTexture );
   m_effect.SetFloatArray("g_StepXY", texSteps, texStepsCount);
-  float colorRange[2] = 
-  { 
-    (useLimitRange ?  16.f / 255.f : 0.f),
-    (useLimitRange ? 219.f / 255.f : 1.f)
-  };
-  m_effect.SetFloatArray("g_colorRange", colorRange, _countof(colorRange));
+  m_effect.SetFloatArray("g_colorRange", getColorRange(useLimitRange), 2);
```

Comment:
To avoid the magic number, how about

```c++
const std::vector<float> &range = getColorRange(useLimitRange);
m_effect.SetFloatArray("g_colorRange", range.data(), range.size());
```